OWNERS

```diff
@@ -0,0 +1,29 @@
+approvers:
```

Comment:
I think we should start using OWNERS_ALIASES along with OWNERS file, 

We already did this in dataplane-operator:
[1]  https://github.com/openstack-k8s-operators/dataplane-operator/blob/main/OWNERS 
[2] https://github.com/openstack-k8s-operators/dataplane-operator/blob/main/OWNERS_ALIASES

Could you please follow dataplane-operator pattern.

Reply:
thanks for the fix.

lgtm now, I will let edpm-approver merge this patch - Incase they want to add someone else in the list.